.travis.yml

```diff
@@ -63,13 +72,13 @@ jobs:
           - /usr/local/Homebrew/Library/Taps
           - $HOME/.cache/pip
     - stage: deploy
-      if: branch = dev AND repo = plus3it/pyppyn
+      if: branch = env(RELEASE_BRANCH) AND tag IS blank
```

Comment:
I'm not sure how to get this condition to really work the way we want... These jobs shouldn't even _appear_ in pull requests. But they do: https://travis-ci.org/plus3it/pyppyn/builds/486620326

Reply:
I think this is worth a try: `if: type != pull_request`

Reply:
They appear but they don't attempt to deploy because it's a PR. The message doesn't always show up but the behavior is consistent.

Reply:
The reason I mention it is because the jobs take _time_ to launch the node and determine that they don't need to do anything. If we can get the jobs to truly be ignored, as the `if` is intended to do, then everything gets faster.